README.rst

```diff
@@ -278,6 +278,16 @@ differ)::
 
     PASSED (skips=15, successes=1322)
 
+
+For the homeserver config, you can copy the sample config:
+
+    cp docs/sample_config.yaml homeserver.yaml
```

Comment:
It looks like there is also a script to generate the config from https://github.com/matrix-org/synapse/blob/develop/INSTALL.md#installing-from-source

```
python -m synapse.app.homeserver \
    --server-name my.domain.name \
    --config-path homeserver.yaml \
    --generate-config \
    --report-stats=[yes|no]
```

But I was following from https://github.com/matrix-org/synapse/#synapse-development

Maybe just need to cross-link instead.